packages/babel-plugin-polyfill-corejs3/src/built-in-definitions.js

```diff
@@ -498,30 +570,69 @@ export const InstanceProperties = {
   concat: define("instance/concat", ["es.array.concat"], undefined, ["String"]),
   copyWithin: define("instance/copy-within", ["es.array.copy-within"]),
   description: define(null, ["es.symbol", "es.symbol.description"]),
+  dotAll: define("instance/dot-all", ["es.regexp.dot-all"]),
+  drop: define("instance/drop", [
+    "esnext.async-iterator.drop",
+    ...AsyncIteratorDependencies,
+    "esnext.iterator.drop",
+    ...IteratorDependencies,
+  ]),
+  emplace: define("instance/emplace", [
+    "esnext.map.emplace",
+    "esnext.weak-map.emplace",
+  ]),
   endsWith: define("instance/ends-with", ["es.string.ends-with"]),
   entries: define("instance/entries", ArrayNatureIteratorsWithTag),
-  every: define("instance/every", ["es.array.every"]),
+  every: define("instance/every", [
+    "es.array.every",
+    "esnext.async-iterator.every",
+    "esnext.iterator.every",
```

Comment:
I think that it's fine to add to methods like this `IteratorDependencies` - it will not add many useless dependencies. Without special configuration, those helpers will not be added to `%AsyncIteratorPrototype%` anyway, so users will be forced to use `AsyncIterator.from` and it will add `AsyncIteratorDependencies`.